src/desktop/alternatives.desktop

```diff
@@ -0,0 +1,17 @@
+[Desktop Entry]
+Type=Application
+Categories=Settings;System;Qt;X-SuSE-YaST;
```

Comment:
well, categories is documented at https://standards.freedesktop.org/menu-spec/latest/apa.html
categories with X- is own specific categories. I think that X-SuSE-YaST-System is deprecated way how to assign categories in ontrol center, which is now obsoleted by X-SuSE-YaST-Group key. But maybe @mvidner knows it for sure.